### PR TITLE
Improve file related error messages

### PR DESCRIPTION
**Pull request by jgberry**
Improves messages in the following cases:
1. A provided file does not exist.
2. A directory was provided but that directory contains no Python files.

Changed behavior:
1. Allow processing of files even if they do not have a `.py` extension.

### CODE REVIEW

File: src/ssort/_main.py [hunk 1]
```diff
@@ -39,7 +39,12 @@ def main():
     for path in find_python_files(args.files):
         errors = False
 
-        original_bytes = path.read_bytes()
+        try:
+            original_bytes = path.read_bytes()
+        except FileNotFoundError:
+            sys.stderr.write(f"ERROR: {str(path)!r} does not exist\n")
+            unsortable += 1
+            continue
```
Comment (bwhmather) [inline src/ssort/_main.py:47]: Probably also doing `IsADirectoryError` and `PermissionError` in the same batch.  I don't think there are any other errors which are expected.  `IsADirectoryError` will be a pain to test as I think it can only result from a race condition.

File: src/ssort/_main.py [hunk 2]
```diff
@@ -157,6 +164,8 @@ def _fmt_count_were(count):
             summary.append(f"{_fmt_count_were(unchanged)} left unchanged")
         if unsortable:
             summary.append(f"{_fmt_count_were(unsortable)} not sortable")
+        if not unsorted and not unchanged and not unsortable:
+            summary.append("No files are present to be sorted. Nothing to do.")
 
         sys.stderr.write(", ".join(summary) + "\n")
```
Comment (bwhmather) [inline src/ssort/_main.py:170]: Please update the tests in `test_executable.py`.

File: src/ssort/_main.py (hunk 1 above)
Comment (jgberry) [reply src/ssort/_main.py:47]: Added. I skipped adding tests for IsADirectoryError, however. I image you could use file locking to consistently test it but the complexity of such a test doesn't seem worth it for such a rare edge case.

File: src/ssort/_main.py (hunk 2 above)
Comment (jgberry) [reply src/ssort/_main.py:178]: Added a test case for this change.

**Review by bwhmather — APPROVED** [review]

**Comment by bwhmather** [comment]
Out of curiosity, are you able to click merge now after I approve?  I'm always a bit uneasy about being the one to hit the merge button on other people's changes on the off chance that they still have more to add.

**Comment by jgberry** [comment]
Yep, I can merge.
